hivemind/p2p/p2p_daemon_bindings/control.py

```diff
@@ -93,6 +103,99 @@ async def listen(self) -> AsyncIterator["ControlClient"]:
         async with server:
             yield self
 
+    async def _read_from_persistent_conn(self, reader: asyncio.StreamReader):
+        while True:
+            resp: p2pd_pb.Response = p2pd_pb.Response()  # type: ignore
```

Comment:
```suggestion
            resp = p2pd_pb.Response()  # type: ignore
```

nit: Seems like obvious type annotation.